testing/logstash/logstash.conf

```diff
@@ -0,0 +1,12 @@
+input {
+        file {
+        type => "syslog"
+        path => ["/var/log/**/*.log"]
+        }
+}
+
+output {
+        elasticsearch { hosts => ["localhost:9200"] }
```

Comment:
this does not seem like a reasonable default as the package does not have a hard dependency on ES